attest/attest_simulated_tpm20_test.go

```diff
@@ -153,6 +153,21 @@ func TestSimTPM20ActivateCredential(t *testing.T) {
 	}
 }
 
+func TestParseAIKPublic(t *testing.T) {
```

Comment:
Could you add a test for TPM 1.2 as well? maybe just parse the public handled in `TestMintAIK` (attest_tpm12_test.go) or something?

Reply:
Added a test. ptal

Reply:
Also, I don't have a good idea how to run these tests. My guess is that it's not part of CI?

Reply:
The CI runs them: https://source.cloud.google.com/results/invocations/c3647263-0de6-4b0d-9a7b-96e3c0c3490a/targets/attest/tests;group=attest;test=TestParseAIKPublic12;row=5

Getting them to run on your machine is witchcraft though: https://github.com/google/go-attestation/blob/master/ci/setup_tpm12_simulator.sh

Reply:
Sweet :)